internal/cli/auth_method_list.go

```diff
@@ -0,0 +1,91 @@
+package cli
```

Comment:
Added `auth-method inspect` (I thought you said "inspect"... I don't care what its called!)

```
$ waypoint auth-method inspect google                                                                                                                                                                                                                        
name: "google"
display_name: "google"
oidc: <
  client_id: "624948239288-fegmqs0q7spv6zujaf53qpvtc70o8h74.apps.googleusercontent.com"
  client_secret: "[REDACTED: client secret]"
  discovery_url: "https://accounts.google.com"
```

Reply:
Nice! I think `inspect` fits better.

The output is broken if you specify a long `issuer-ca-pem` (and I imagine it will usually be long?):

```
$ ./waypoint auth-method set oidc -client-id=long-test-client-id -client-secret=redacted-secret -issuer=https://gitlab.com -issuer-ca-pem="AAAAB3NzaC1yc2EAAAADAQABAAACAQDrTXy4An3csJT0X44P8qBu2Ci5fYWHbWXaTgSeWlT0RYmKA7cYjMxTMyK5QOsVNplu2IHbmFGpQBX3004dJ35Ors9GrH+rUDhF53MTV8Suyt8aifGPPq+eeAaL3cSEnjqpt5JN+buOL/3tBVxVzx5ghxACzvP4OICBFCu2n4qJR198bJvhTI9EGKIZzkIdEvZvPoP9dawG+Dv3HlXs2lFxf3k3vAZ0SeX0UoobAEKZEBVj4NyhnkUdvSAlXd1KpTQzmfnq/1VYdfBqUorNfpABn9h/GWjv4PBpakFhy8xwyhz0g1FD1A6+uSmviX2/EN6I1GXS4M8amuFuW74wWy/pvl1I6DSPSi4G5ROD4azSZSXwxm10J53qsgzio+rpr8w6F7HNMGKNQ8PbFfV86BxlNvUmjG1e/akVW4ygHI7zmusGq+8NnOx/Ws72cPmR6ZP+Y5JMr/b9ttSPaYSmYs75wMZfqUwI1YlQRugdIqjOyaFlKBkPpFCQkRxFkU9FmxuedOLDtpTjOgyZ5mycCwI6ifh3hIBTxyYvozN1peWz4oKwfc/dP04v97O+tAU9w2GEu5/X9faMk+23zkb1Oqps3Ecvfh51F/3Qtxp+bgX+75QoqPRf1nftiDKCFZm4tkx8W6Qpo69rEdX1xtQbEsl1Y8ujtjj+WgjUu495hfDsFQ==" -claim-scope=foo -claim-scope=bar -claim-scope=baz -allowed-redirect-uri=foobar.baz -signing-algorithm=secure -signing-algorithm=insecure oidc-test
Auth method configured.

$ ./waypoint auth-method inspect oidc-test
name: "oidc-test"
display_name: "oidc-test"
oidc: <
  client_id: "long-test-client-id"
  client_secret: "[REDACTED: client secret]"
  scopes: "foo"
  scopes: "bar"
  scopes: "baz"
  allowed_redirect_uris: "foobar.baz"
  signing_algs: "secure"
  signing_algs: "insecure"
  discovery_url: "https://gitlab.com"
  disco
"AAAAB3NzaC1yc2EAAAADAQABAAACAQDrTXy4An3csJT0X44P8qBu2Ci5fYWHbWXaTgSeWlT0RYmKA7cYjMxTMyK5QOsVNplu2IHbmFGpQBX3004dJ35Ors9GrH+rUDhF53MTV8Suyt8aifGPPq+eeAaL3cSEnjqpt5
>
```

Probably an unrelated glint bug, but just noting.

Reply:
Yeah, I'll just use `fmt` for now then.

Reply:
Also kind of weird that you set the `-issuer` flag, but it comes back as `discovery_url`. Same with `-issuer-ca-pem` becoming `discovery_ca_pem`.

Reply:
yeah, agreed, I think `discovery-url` is more descriptive but the industry standard term seems to be `issuer` for this.